docs/guides/sticky/README.md

```diff
@@ -17,7 +17,10 @@ _setRef(recycler) {
 * If using `overrideRowRenderer`, keep in mind that upon scrolling to the very top or bottom of the content, stickies will be hidden. eg. If the first item in the list is given as sticky, scrolling to the top will display the original view and not the overridden view.
 
 ### 2) Props
-* `stickyHeaderIndices`   - An array of indices whose corresponding items need to be stuck to the top of the RecyclerListView once the items scroll off the top. Every subsequent sticky index view will push the previous sticky view off the top to take its place. Needs to be sorted ascending.
-* `stickyFooterIndices`   - Works same as sticky headers, but for views to be stuck at the bottom of the recyclerView. Needs to be sorted ascending.
-* `overrideRowRenderer`   - Optional. Will be called instead of rowRenderer for all sticky items. Any changes to the item for when they are stuck can be done here. Refer to sample code for usage.
-* `style`                 - Optional. Pass the same style that is applied to the RecyclerListView component here.
+* `stickyHeaderIndices`     - An array of indices whose corresponding items need to be stuck to the top of the RecyclerListView once the items scroll off the top. Every subsequent sticky index view will push the previous sticky view off the top to take its place. Needs to be sorted ascending.
+* `stickyFooterIndices`     - Works same as sticky headers, but for views to be stuck at the bottom of the recyclerView. Needs to be sorted ascending.
+* `overrideRowRenderer`     - Optional. Will be called instead of rowRenderer for all sticky items. Any changes to the item for when they are stuck can be done here. Refer to sample code for usage.
+* `stickyContainerRenderer` - Optional. Pass a stylized container for StickyHeader and StickyFooter, providing user extensibility to customize the look and feel of these items.
+* `scrollOffsetCorrection`  - Optional. Will be called upon onScroll of the RecyclerListView, and scroll offset will be corrected based on the value returned from this function.
```

Comment:
Name doesn't sound right. You should call it getLogicalOffset and define logical offset and it's impact and how it is different from actual opne